install/install_requirements.sh

```diff
@@ -106,3 +106,5 @@ fi
   set -x
   $PIP_EXECUTABLE install evaluate=="0.4.3" lm-eval=="0.4.2" psutil=="6.0.0"
 )
+
+export HF_HUB_ENABLE_HF_TRANSFER=1
```

Comment:
I don't think this will do anything unless people are running `install_requirements.sh` via `source`, which would be unusual.